packages/documentation-site/patternfly-docs/content/design-guidelines/charts/about/about.md

```diff
@@ -65,3 +65,15 @@ On click
 
 1. **Legend:** When a user clicks on a legend label, it becomes disabled and the color swatch is replaced with an eye-slashed icon.
 2. **Chart:** Data corresponding to the clicked legend label is hidden from view.
+
+## Develop with charts 
+
+Default styles in the [@patternfly/react-charts package](https://www.npmjs.com/package/@patternfly/react-charts) are aligned with our light theme. To use chart styles, you must also have the [@patternfly/patternfly package](https://www.npmjs.com/package/@patternfly/patternfly) installed.
+
+To utilize default chart styles, you don't need to import anything else. But, to support dark-themed charts, you must also import the stylesheet that contains dark theme styles. To do so, add this line before importing your main application component:
```

Comment:
Also confused about "utilize default chart styles" - not sure what "utilize" means exactly. Same with "to support dark-themed charts" - not sure what "support" means. If I simplifed it, I'd say charts works with light theme out of the box, no additional styles or anything are needed. However, for charts to work in dark theme, you'll need to import `@patternfly/patternfly/patternfly-charts.css`

Reply:
Just to clarify, when you say "global tokens", do you mean design tokens? Or [this file](https://github.com/patternfly/patternfly/blob/main/src/patternfly/base/tokens/tokens-default.scss)? Which, this file is ultimately our design tokens..right?  

I should find time to create a cheat sheet for some of these terms React tokens, design tokens, variables, styles, etc. I get so mixed up 😅

Reply:
Your simplified suggestion makes sense outside of me making sure I understand the verbiage, though! Gonna make updates based off this